include/zephyr/bluetooth/classic/hfp_hf.h

```diff
@@ -53,108 +40,380 @@ struct bt_hfp_hf_cb {
 	 *  connection completes.
 	 *
 	 *  @param conn Connection object.
+	 *  @param hf HFP HF object.
 	 */
-	void (*connected)(struct bt_conn *conn);
+	void (*connected)(struct bt_conn *conn, struct bt_hfp_hf *hf);
 	/** HF disconnected callback to application
 	 *
 	 *  If this callback is provided it will be called whenever the
 	 *  connection gets disconnected, including when a connection gets
 	 *  rejected or cancelled or any error in SLC establishment.
 	 *
-	 *  @param conn Connection object.
+	 *  @param hf HFP HF object.
 	 */
-	void (*disconnected)(struct bt_conn *conn);
+	void (*disconnected)(struct bt_hfp_hf *hf);
```

Comment:
This seems to be missing documentation that the object is considered invalid/freed after this callback returns.

Reply:
Updated.